<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display</class>
 <widget class="QMainWindow" name="display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>display</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>376</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/plantilla.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_set">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>236</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/set.png</normaloff>:/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_menos">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>236</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/menos.png</normaloff>:/menos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mas">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>236</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/mas.png</normaloff>:/mas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>236</y>
      <width>37</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/start.png</normaloff>:/start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 27, 27);
color: rgb(171, 0, 0);</string>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLabel" name="texto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 136);
background-color: rgb(168, 168, 168);
</string>
    </property>
    <property name="text">
     <string>Apagado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>120</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/switch_off.png</normaloff>:/switch_off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_temperatura">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>240</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/cf.png</normaloff>:/cf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_pantalla">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledAviso">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>88</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledAviso_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pipipi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 31, 32);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ledSet">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>88</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledSet_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ledCalor">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>88</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledCalor_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ledTimer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>88</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledTimer_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ledTempC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>106</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledC_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ledTempF">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>134</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/ledF_Off.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_agua">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>50</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AGUA</string>
    </property>
   </widget>
   <widget class="QLabel" name="llenarAgua">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>115</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>115</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/aviso/agua0.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;** NOTA **&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;En esta versión no está incluido las siguientes funciones:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- Contar los minutos en negativo cuando termina el ciclo&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;- modificación de los parámetros cuando está dentro del ciclo con el SET&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>fondo_pantalla</zorder>
   <zorder>background</zorder>
   <zorder>button_set</zorder>
   <zorder>button_menos</zorder>
   <zorder>button_mas</zorder>
   <zorder>texto</zorder>
   <zorder>button_power</zorder>
   <zorder>button_temperatura</zorder>
   <zorder>ledAviso</zorder>
   <zorder>ledSet</zorder>
   <zorder>ledCalor</zorder>
   <zorder>ledTimer</zorder>
   <zorder>ledTempC</zorder>
   <zorder>ledTempF</zorder>
   <zorder>pipipi</zorder>
   <zorder>button_agua</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>llenarAgua</zorder>
   <zorder>textEdit</zorder>
   <zorder>button_start</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
